service.ts

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+import crypto from 'node:crypto';
+import bodyParser from 'body-parser';
+import { createValidator } from 'express-joi-validation';
+import {
+  createUserBodySchema,
+  paramsIDSchema,
+  paramsSubstringLimitSchema,
+  updateUserBodySchema,
+} from './schemas';
+import type { User } from './types';
+
+const app = express();
+const validator = createValidator();
+
+app.listen(3000);
```

Comment:
This is the last step, actually. First you add all your routs, then you start server.

Reply:
Ans also that won't be `service`, that's `app` file or `server` file. If you create a separate file for router (you can read about it in express doc), then `router` file.

Reply:
fixed